cmd/queue/main_test.go

```diff
@@ -441,3 +441,11 @@ func TestQueueTraceSpans(t *testing.T) {
 		})
 	}
 }
+
+func NewTestServer(f func(w http.ResponseWriter)) *httptest.Server {
```

Comment:
Why is this public? It also seems to be a `newProbeTestServer`.

Reply:
Done.